### PR TITLE
[webui] Kiwi templates

### PR DESCRIPTION
**Pull request by Ana06**
Implementation of a SUSE Studio like image selection page

### CODE REVIEW

**Comment by Ana06** [comment]
Can we squash 	ec9f04f and 	f359574 into 	7b478cf ?

**Comment by Ana06** [comment]
[(8) Present kiwi templates](https://trello.com/c/0JyzpRkk/112-8-present-kiwi-templates)

**Comment by bgeuken** [comment]
@Ana06 yes, please do so

**Comment by mschnitzer** [comment]
@Ana06 since you picked eduardos branch, please double check if nothing is missing from this branch: https://github.com/hennevogel/open-build-service/commits/ChrisBr-branching_image_templates (might be possible cause I already posted one commit what's not in there)

File: src/api/app/views/webui/main/index.html.erb [hunk 1]
```diff
@@ -22,6 +22,7 @@
       <% unless @spider_bot -%>
           <%= proceed_link 'utilities-system-monitor', 'Status Monitor', :controller => 'monitor', :action => 'index' %>
       <% end -%>
+      <%= proceed_link 'distributions-kiwi', 'New Image', image_templates_path unless User.current.is_nobody? %>
```
Comment (hennevogel) [inline src/api/app/views/webui/main/index.html.erb]: This should be 'hidden' by now. It was in my branch

**Comment by adrianschroeter** [comment]
On Freitag, 25. November 2016, 02:27:58 CET wrote Henne Vogelsang:
> hennevogel requested changes on this pull request.
> 
> 
> 
> > @@ -22,6 +22,7 @@
>        <% unless @spider_bot -%>
>            <%= proceed_link 'utilities-system-monitor', 'Status Monitor', :controller => 'monitor', :action => 'index' %>
>        <% end -%>
> +      <%= proceed_link 'distributions-kiwi', 'New Image', image_templates_path unless User.current.is_nobody? %>
> 
> This should be 'hidden' by now. It was in my branch

Maybe just enable it for admins for now, so we can still see it in test installations.

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany

File: src/api/app/views/webui/main/index.html.erb (hunk 1 above)
Comment (Ana06) [inline src/api/app/views/webui/main/index.html.erb]: I removed it accidentally.

**Comment by Ana06** [comment]
So I keep it for admins then

**Comment by Ana06** [comment]
> @Ana06 since you picked eduardos branch, please double check if nothing is missing from this branch: https://github.com/hennevogel/open-build-service/commits/ChrisBr-branching_image_templates (might be possible cause I already posted one commit what's not in there)

2b41c4b40f3012f69789a5485a367aaa5fb70714 was not in https://github.com/hennevogel/open-build-service/commits/ChrisBr-branching_image_templates neither

I'll add it anyway. Although I think we can also squash it too into 7b478cf

**Comment by hennevogel** [comment]
Alright here we go!

**Pull request merged**